spec/serializers/v1/invoice_serializer_spec.rb

```diff
@@ -25,12 +25,14 @@
       expect(result['invoice']['amount_cents']).to eq(invoice.amount_cents)
       expect(result['invoice']['amount_currency']).to eq(invoice.amount_currency)
       expect(result['invoice']['vat_amount_cents']).to eq(invoice.vat_amount_cents)
+      expect(result['invoice']['sub_total_vat_excluded_amount_cents']).to eq(invoice.sub_total_vat_excluded_amount_cents)
```

Comment:
Layout/LineLength: Line is too long. [121/120] (https://rubystyle.guide#max-line-length)